server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1503,10 +1503,17 @@ private void blockForTservers() throws InterruptedException {
     // yield to happen
     long initialWait = Math.min(50, maxWait / 2);
 
-    Retry tserverRetry =
-        Retry.builder().infiniteRetries().retryAfter(initialWait, TimeUnit.MILLISECONDS)
-            .incrementBy(15_000, TimeUnit.MILLISECONDS).maxWait(maxWait, TimeUnit.MILLISECONDS)
-            .logInterval(30_000, TimeUnit.MILLISECONDS).createRetry();
+    long logInterval = 30_000;
+    long timeIncrement = 15_000;
+    // maxWait is the maximum amount of time for retry commands to be attempted.
+    // Let's attempt the max amount of retries in the provided time duration.
+    long maxRetries = Math.round((double) (maxWait - initialWait) / (timeIncrement + logInterval));
```

Comment:
I expect that log interval should not be part of the calculation,  Log interval is used to prevent spamming the logs with retry statements - say you have a 100ms timeout but want to wait 10s, you don't gain much from 10,000s of log staements saying retrying....  in that case, a log every few seconds so that you know its not dead is sufficient.